<ui version="4.0" >
 <class>LayoutOptionWidget</class>
 <widget class="QWidget" name="LayoutOptionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Text layout engine options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Weights</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QSlider" name="beforeSlid" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="tracking" >
           <bool>true</bool>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition" >
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3" >
         <widget class="QLabel" name="bScore" >
          <property name="text" >
           <string>score</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>beforeSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Before</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="1" >
         <widget class="QSlider" name="fitSlid" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="tracking" >
           <bool>true</bool>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3" >
         <widget class="QLabel" name="eScore" >
          <property name="text" >
           <string>score</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>fitSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Exact</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="2" >
       <layout class="QGridLayout" name="gridLayout_4" >
        <item row="0" column="1" >
         <widget class="QSlider" name="afterSlid" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="tracking" >
           <bool>true</bool>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_6" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3" >
         <widget class="QLabel" name="aScore" >
          <property name="text" >
           <string>score</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>afterSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>After</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3" >
       <layout class="QGridLayout" name="gridLayout_5" >
        <item row="0" column="1" >
         <widget class="QSlider" name="endSlid" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="tracking" >
           <bool>true</bool>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_7" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_8" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3" >
         <widget class="QLabel" name="endScore" >
          <property name="text" >
           <string>score</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>endSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>End</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4" >
       <layout class="QGridLayout" name="gridLayout_6" >
        <item row="0" column="1" >
         <widget class="QSlider" name="hyphSlid" >
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="tracking" >
           <bool>true</bool>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_9" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_10" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3" >
         <widget class="QLabel" name="hPenalty" >
          <property name="text" >
           <string>penalty</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>hyphSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>H-penalty</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="5" >
       <layout class="QGridLayout" name="gridLayout_7" >
        <item row="0" column="1" >
         <widget class="QSlider" name="spaceSlid" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>100</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_11" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" >
         <spacer name="horizontalSpacer_12" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4" >
         <widget class="QLabel" name="minSpace" >
          <property name="text" >
           <string>space</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>spaceSlid</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Max. compression</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
